content/en/user/posting.md

```diff
@@ -121,7 +121,7 @@ Send your post only to mentioned users.
 * Your post cannot be boosted.
 
 {{< hint style="warning" >}}
-**Do not share dangerous and sensitive information over direct messages**. Mastodon is not an encrypted messaging app like Signal or Wire, the database administrators of the sender’s and recipient’s servers have access to the text. Use them with the same caution as you would use forum PMs, Discord PMs and Twitter DMs.
+**Do not share dangerous and sensitive information over private mentions**. Mastodon is not an encrypted messaging app like Signal, Matrix or Wire. By accessing the Mastodon database directly the administrators of your server, and any recipient’s servers, can read your messages.
```

Comment:
`By accessing the Mastodon database directly the administrators of your server, and any recipient’s servers, can read your messages.` -> `The administrators of your server and any receiving servers can theoretically read your posts by directly accessing their Mastodon server's database.`

Reply:
agreed that this is a better phrasing (and avoids starting a sentence with a preposition "By", which is grammatically fine, but I personally would try to avoid in technical writing)

Reply:
I agree that this phrasing is more direct, honest and doesn't cause as much fear as the previous phrasing does.